fedimint-core/src/module/mod.rs

```diff
@@ -400,7 +396,15 @@ impl ApiEndpoint<()> {
                         .to_typed()
                         .map_err(|e| ApiError::bad_request(e.to_string()))?;
 
-                    let ret = handle_request::<E>(m, &mut context, request).await?;
+                    let span = tracing::info_span!(
+                        target: LOG_NET_API,
+                        "api_req",
+                        id = REQ_ID.fetch_add(1, Ordering::SeqCst),
```

Comment:
Does JSON RPC allow sending some connection identifier? Would be great if we could track these requests across the gateway/client <> federation boundary (I think that's how distributed tracing is supposed to work?).

Reply:
The client picks its own request ID and server responds with it.

But I'm not aware of any place to put a server-side ID in the response.